recipe/meta.yaml

```diff
@@ -14,35 +14,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Orange3-{{ version }}.tar.gz
-  sha256: d436bb98d33aa6ea7b6083359032c704da1a6a0e656c068ce806e2a3f319af8c
-  patches:
-    - patches/0001-setup.py-Replace-use-of-imp-module.patch
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 6c6d3fa82e01fdc16187ef027cebb91537eae5b129ba31d3bb4bbfdab5fbb4f4
 
 build:
   number: 0
   entry_points:
     - orange-canvas = Orange.canvas.__main__:main
   osx_is_app: true
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
-  skip: True  # [linux and (s390x or ppc64le)]
+  skip: True  # [py<39 or s390x]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch   # [not win]
-    - m2-patch    # [win]
   host:
     - pip
     - python
-    - cython >=3.0
```

Comment:
This matches upstream, so I think it's better to keep it as is.

Reply:
Or just switch to `cython 3.0` and do the same for the other host dependencies

Reply:
Thanks Steve! Made a few tweaks!